client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-const MovieCard = props => {
-  return;
+const MovieCard = ({ movie }) => {
```

Comment:
Refactoring the project to apply this component in multiple places is probably the best example of re-usability of React components.